src/servers/launcher/main.cpp

```diff
@@ -0,0 +1,849 @@
+#include <libpq-fe.h>
+
+#include <array>
+#include <cerrno>
+#include <chrono>
+#include <cstdint>
+#include <cstring>
+#include <expected>
+#include <iomanip>
+#include <iostream>
+#include <netinet/in.h>
+#include <ranges>
+#include <signal.h>
+#include <stdexcept>
+#include <sstream>
+#include <string>
+#include <sys/epoll.h>
+#include <sys/wait.h>
+#include <thread>
+#include <unistd.h>
+#include <utility>
+#include <variant>
+
+using namespace std::literals;
+using notManyMilliseconds = std::chrono::duration<int, std::milli>;
+using PacketBuffer = std::array<uint8_t, 65536>;
+
+static constexpr const auto RETRY_DELAY{1s};
+static constexpr const uint16_t PORT{32232};
+static constexpr const uint8_t CABOOSE{108};
+
+struct UUID {
+  std::array<const uint8_t, 16> uuid;
```

Comment:
In any case, [you should never make data members `const`, especially in a value type like this that should, by all rights, be copyable and movable](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c12-dont-make-data-members-const-or-references-in-a-copyable-or-movable-type).

Reply:
I...  Kinda liked that it was move-only.  It meant my compiler shouted at me whenever I tried to copy it when I never intended to.  It was a good learning experience.

I don't disagree that the type should be copyable (it's just 128 bits), but I really liked the kick in the pants.  Can you suggest any alternative guardrails to alert me to inadvertant copies?